doc/api/async_hooks.md

```diff
@@ -730,9 +732,9 @@ fs.open(path, 'r', (err, fd) => {
 ```
 
 ```cjs
-const async_hooks = require('node:async_hooks');
+const { executionAsyncId } = require('node:async_hooks');
 
-console.log(async_hooks.executionAsyncId());  // 1 - bootstrap
+console.log(executionAsyncId());  // 1 - bootstrap
 fs.open(path, 'r', (err, fd) => {
   console.log(async_hooks.executionAsyncId());  // 6 - open()
```

Comment:
Missed one?

Reply:
well spotted 👍

Reply:
> Unlike the other PR, I'm mixed on this one. There's a lot of inconsistency in the examples that go beyond what you've fixed here so I'm not sure this is an improvement to the overall doc quality.

Fair point. I guess what I "tried" to improve is to keep `mjs` and `cjs` import consistent in the example? e.g. if `mjs` use es6 destructing syntax, then `cjs` probably should follow it?

Reply:
added missing `fs` import to both `mjs`, `cjs`